frontend/src/index.css

```
@@ -1,68 +1,569 @@
+}
+:host,
+html {
+  line-height: 1.5;
+  -webkit-text-size-adjust: 100%;
+  font-family:
+    Open Sans,
+    ui-sans-serif,
+    system-ui,
+    sans-serif,
+    Apple Color Emoji,
+    Segoe UI Emoji,
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Emoji

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)